pkg/terraformer/types.go

```diff
@@ -63,6 +64,12 @@ type terraformer struct {
 	deadlinePod      time.Duration
 }
 
+// RawState represent the terraformer state's raw data
+type RawState struct {
+	Data   string `json:"data"`
+	Encode string `json:"encode"`
```

Comment:
`encode` is a verb, can we change to `encoding`? (as the coreos-cloudinit, the have similar `encoding` field)

```suggestion
	Encoding string `json:"encoding"`
```